addons/binding/org.openhab.binding.opensprinkler/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,501 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="opensprinkler"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- The stand alone OpenSprinkler thing type -->
+    <thing-type id="opensprinkler">
+        <label>OpenSprinkler Thing</label>
+        <description>This is the stand alone OpenSprinkler device that allows sprinkler control.</description>
+
+        <channels>
+            <channel id="rainsensor" typeId="rainsensor" />
+            <channel id="station01" typeId="station01" />

```

Comment:
I think all of your stations could be of the same channel-type, right? Note that you can give (by now, this was probably not yet in place when you implemented the binding) individual labels and descriptions per channel, so these are not inherited from the channel-type. So I would suggest something like:

```
            <channel id="01" typeId="station">
		        <label>Station 01</label>
		        <description>Controls station 01 on the OpenSprinkler device.</description>
            </channel>
            <channel id="02" typeId="station">
		        <label>Station 02</label>
		        <description>Controls station 02 on the OpenSprinkler device.</description>
            </channel>
...
    <channel-type id="station">
        <item-type>Switch</item-type>
        <label>Station</label>
        <description>Controls a station on the OpenSprinkler device.</description>
        <category>Switch</category>
    </channel-type>
```